### PR TITLE
Mydevelopment

### PR DESCRIPTION
**Pull request by tigerisbigcat**
add some features to the home page.

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,388 @@
...
+  margin-top: -95%;
+  margin-left: 2700%;
+}
+
+.dot1 {
+  width: 25px;
+  height: 25px;
+  background-color: #b5bbc6;
+}
+*/
```
Comment (geegeex) [inline portfolio/src/main/webapp/style.css:247]: It is best practice to not leave uncommented code in.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,388 @@
+body {
+  margin: 0;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+/* menu settings */
```
Comment (geegeex) [inline portfolio/src/main/webapp/style.css:6]: Good job with the comments! It makes code more legible!

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +26,29 @@ function addRandomGreeting() {
...
+    const responseFromServer = await fetch('/form-handler');
+
+    // Parse the response as JSON.
+    // const myObject = await responseFromServer.text() + "test";
+    const textFromResponse = await responseFromServer.text() + "test";
+
+    const dateContainer = document.getElementById('date-container-json');
+    dateContainer.innerText = textFromResponse;
+
+    // // Now we can reference the fields in myObject!
```
Comment (geegeex) [inline portfolio/src/main/webapp/script.js:50]: Remove commented out code.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -26,3 +26,29 @@ function addRandomGreeting() {
...
+    dateContainer.innerText = textFromResponse;
+  }
+
+async function jsonFile() {
+    // Send a request to /my-data-url.
+    const responseFromServer = await fetch('/form-handler');
+
+    // Parse the response as JSON.
+    // const myObject = await responseFromServer.text() + "test";
+    const textFromResponse = await responseFromServer.text() + "test";
```
Comment (geegeex) [inline portfolio/src/main/webapp/script.js:45]: Try to remove test fields from the response body ("test").

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -7,13 +7,333 @@
...
+        </div>
+
+        <div class="contact_form_setting">
+          <form action="/form-handler" method="POST">
+            <input class="form_control" name="name" type="text" placeholder="Your Name">
+            <input class="form_control" name="email" type="email" placeholder="Your Email">
+            <input class="form_control" name="phone" type="phone" placeholder="Phone Number">
+            <textarea class="form_control" name="message" style="height: 100px;" placeholder="Your Message"></textarea>
+            <button class="form_control" id="submit" type="submit">SEND</button>
+            <!-- <div id="success">
```
Comment (geegeex) [inline portfolio/src/main/webapp/index.html:305]: Remove unused code.

**Review by geegeex — CHANGES_REQUESTED** [review]
Great job with the pull request! Left some comments. Please reach out if you have any questions!

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (tigerisbigcat) [reply portfolio/src/main/webapp/style.css:247]: test

Comment (tigerisbigcat) [reply portfolio/src/main/webapp/style.css:247]: I deleted it. Done.

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (tigerisbigcat) [reply portfolio/src/main/webapp/script.js:50]: Done.

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (tigerisbigcat) [reply portfolio/src/main/webapp/script.js:45]: Done.

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (tigerisbigcat) [reply portfolio/src/main/webapp/index.html:305]: Done.

**Review by geegeex — APPROVED** [review]

**Pull request merged**